src/ethereum_spec_tools/evm_tools/fixture_loader.py

```diff
@@ -291,9 +299,14 @@ def json_to_tx(self, raw: Any) -> Any:
             parameters.insert(
                 8, self.json_to_access_list(raw.get("accessList"))
             )
-            return b"\x02" + rlp.encode(
-                self._module("fork_types").FeeMarketTransaction(*parameters)
-            )
+            try:
+                return b"\x02" + rlp.encode(
+                    self._module("fork_types").FeeMarketTransaction(
+                        *parameters
+                    )
+                )
+            except AttributeError as e:
+                raise UnsupportedTx(b"\x02" + rlp.encode(parameters), str(e))
```

Comment:
Unless there's a good reason to discard the original error, I'd recommend:

```suggestion
                raise UnsupportedTx(b"\x02" + rlp.encode(parameters)) from e
```